vendor/qrcode.js

```diff
@@ -1785,11 +1786,19 @@ var qrcode = function() {
     }
     base64.flush();
 
+    var dataUri = '';
+    dataUri += 'data:image/gif;base64,';
+    dataUri += base64;
+    return dataUri;
+  };
+
+
+  var createImgTag = function(width, height, getPixel, alt) {
     var img = '';
     img += '<img';
     img += '\u0020src="';
     img += 'data:image/gif;base64,';
```

Comment:
Do we need this line? Isn't it duplicated as in CreateDataUri we have it as well?